scripts/ctool

```diff
@@ -969,11 +973,17 @@ delete_container() {
     lxc delete --force "$1"
 }
 
+while [ "${1:-}" = "-v" -o "${1:-}" = "--verbose" ]; do
```

Comment:
This relies on `-v` to be the only global option so far, right?

Should we modify the Usage line in `Usage_main()` to something like:
```
Usage: ${0##*/} [global option] mode [options]
```
?

Reply:
If you want i can do that.  global options are also accepted by most of those commands (execute is currently wierd).

Reply:
Well, I think it would be nice to document the fact that global options can go before the command.